app/src/components/Statistics/StatisticsElement.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export const StatisticsElement = ({title, description, icon}) => {
+  return (
+    <Element>
+        <img src={icon} alt="Icon"/>
+        <h2>{title}</h2>
+        <p>{description}</p>
+    </Element>
+  )
+}
+
+const Element = styled.div`
```

Comment:
Element name is not correct. Maybe StatisticsBox or something like that ? :)

Reply:
oh, one more thing. Move it to separate file

Reply:
corrected. I will work on statistics components and others. I just wanted to send PR to work on everything before working further on building site.